### PR TITLE
Relate repository changes + add php samples

### PR DESCRIPTION
**Pull request by stafyniaksacha**
* Add PHP Samples
* Add policies/roles changes
* Add multiprofiles changes
* Remove hydratation methods/options

### CODE REVIEW

File: source/includes/security/_kuzzleUser.md [hunk 1]
```diff
@@ -623,14 +649,14 @@ Updating an user will have no impact until the <code>save</code> method is calle
 
 Replace the profile associated to the user
 
-### setProfile(profileId)
+### setProfiles(profilesIds)
 
-### setProfile(kuzzleProfile)
+### setProfiles(kuzzleProfile)
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: hum you sure ?

File: source/includes/security/_kuzzleUser.md [hunk 2]
```diff
@@ -513,40 +485,57 @@ Resolves to a `KuzzleUser` object.
...
 };
 [user setContentWithData: newContent];
 ```
 
 ```swift
 let newContent = [
-  "profile": [
-    "anotherProfileId"
+  "profilesIds": [
+    "anotherprofilesIds"
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: sugestion: replace with "aProfileId"

File: source/includes/security/_kuzzleUser.md [hunk 3]
```diff
@@ -513,40 +485,57 @@ Resolves to a `KuzzleUser` object.
 ```js
 var user = kuzzle.security.getUser('myuser');
 var userContent = {
-  // define new users properties
+  profilesIds: ['anotherprofilesIds']
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: sugestion: replace with "aProfileId"

File: source/includes/security/_kuzzleUser.md [hunk 4]
```diff
@@ -513,40 +485,57 @@ Resolves to a `KuzzleUser` object.
...
 
 user = user.setContent(userContent);
 ```
 
 ```java
 JSONObject newContent = new JSONObject()
-  .put("profile", new JSONArray()
-    .put("another profile id")
+  .put("profilesIds", new JSONArray()
+    .put("anotherprofilesIds")
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: ditto

File: source/includes/security/_kuzzleUser.md [hunk 5]
```diff
@@ -513,40 +485,57 @@ Resolves to a `KuzzleUser` object.
...
 
 user.setContent(newContent);
 ```
 
 ```objective_c
 NSDictionary* newContent = @{
-  @"profile": @[
-    @"anotherProfileId"
+  @"profilesIds": @[
+    @"anotherprofilesIds"
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: ditto

File: source/includes/security/_kuzzleUser.md (hunk 2 above)
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: ditto

File: source/includes/security/_kuzzleUser.md [hunk 6]
```diff
@@ -513,40 +485,57 @@ Resolves to a `KuzzleUser` object.
...
 
+```php
+<?php
+
+use Kuzzle\Security\User;
+
+// ...
+
+$userContent = [
+  'profilesIds' => ['anotherprofilesIds']
```
Comment (j33f) [inline source/includes/security/_kuzzleUser.md]: ditto

File: source/includes/security/_kuzzleProfile.md [hunk 7]
```diff
@@ -93,17 +128,25 @@ Returns to the `KuzzleProfile` object.
...
 | `id` | string | Unique profile identifier | get |
 
-## addRole
+## addPolicy
 
 ```js
-// Passing a KuzzleRole object
-profile.addRole(kuzzle.security.roleFactory("some role ID", {}));
+// Passing a KuzzlePolicy object
+var policy = profile.policyFactory([
```
Comment (ballinette) [inline source/includes/security/_kuzzleProfile.md]: use ```{ ... }``` instead of ```[ ... ]```

File: source/includes/security/_kuzzleProfile.md [hunk 8]
```diff
@@ -93,17 +128,25 @@ Returns to the `KuzzleProfile` object.
...
+  'roleId': 'some role id',
+  'restrictedTo': [{index: 'index1'}, {index: 'index2', collections: ['foo', 'bar'] } ]
+])
+profile.addPolicy(policy);
 
-// Or by passing its id
-profile.addRole('myrole');
+// Or by passing its description
+profile.addPolicy([
```
Comment (ballinette) [inline source/includes/security/_kuzzleProfile.md]: ditto

File: source/includes/security/_kuzzleProfile.md [hunk 9]
```diff
@@ -659,67 +680,73 @@ Replaces the content of the `KuzzleProfile` object.
...
+## setPolicies
 
 <aside class="note">
 Updating a profile will have no impact until the <code>save</code> method is called
 </aside>
 
 ```js
-KuzzleRole role = kuzzle.security.roleFactory("roleId", {});
+policy = profile.policyFactory([
+  "roleId": "roleId"
```
Comment (ballinette) [inline source/includes/security/_kuzzleProfile.md]: ditto

**Comment by ballinette** [comment]
+1

**Pull request merged**